{{cookiecutter.project_slug}}/.flake8

```diff
@@ -19,4 +19,4 @@ ignore =
     E402
     # already covered by PyLint and gives false positives for typing.overload
     F811
-max-line-length = 100
+max-line-length = 300
```

Comment:
Please add a comment here and in `.pylintrc` that we use a large value because we let `black` handle the line length.

Reply:
Added